### PR TITLE
Release note about node.js adapter and javascript adapter released in…

### LINKED ISSUE #35856: Release note about node.js adapter and javascript adapter released independently of keycloak server
### Description

It can be good to include in Keycloak 26.1 release notes the fact that node.js and javascript adapters will be decoupled from Keycloak server release lifecycle after Keycloak 26.1 release. If we don't announce it 26.1 release notes, it might be strange that adapters will disappear during Keycloak 26.1.Z server micro-releases (if we decouple them from release cycle after Keycloak 26.1 release). As the release notes for Keycloak 26.2 will happen much late in spring.

### Discussion

_No response_

### Motivation

_No response_

### Details

_No response_

### PR DESCRIPTION
**Pull request by mposolda**
…dependently of keycloak server

closes #35856

### CODE REVIEW

File: docs/documentation/release_notes/topics/26_1_0.adoc [hunk 1]
```diff
@@ -11,6 +11,11 @@ darkMode=false
 
 Alternatively, you can disable dark mode support for the built-in Keycloak themes on a per-realm basis by turning off the "Dark mode" setting under the "Theme" tab in the realm settings.
 
+= Dedicated release cycle for the Node.js adapter and Javascript adapter
+
+From this release, the {project_name} javascript adapter and {project_name} Node.js adapter will have release cycle independent of the {project_name} server release cycle. The 26.1.0 release may be the last one
+when those adapters are released together with the {project_name} server. But from now on, the adapters may be released at a different time than the {project_name} server.
```
Comment (jonkoops) [inline docs/documentation/release_notes/topics/26_1_0.adoc:17]:
```suggestion
= Dedicated release cycle for the Node.js adapter and JavaScript adapter

From this release onwards, the {project_name} JavaScript adapter and {project_name} Node.js adapter will have a release cycle independent of the {project_name} server release cycle. The 26.1.0 release may be the last one
where these adapters are released together with the {project_name} server, but from now on, these adapters may be released at a different time than the {project_name} server.
```

**Review by jonkoops — DISMISSED** [review]
Couple of suggestions, but LGTM

**Review by andymunro — APPROVED** [review]
Looks good, @mposolda

**Comment by mposolda** [comment]
@jonkoops @andymunro Thanks for the review!

**Pull request merged**